test/export/test_export.py

```diff
@@ -169,12 +168,24 @@ def foo_unbacked(x):
 
 
 @dataclass
-class Inp:
+class Inp1:
     x: Tensor
     y: List[Tensor]
     z: Dict[str, Tensor]
 
 
+@dataclass
+class Inp2:
+    a: Tensor
+    b: Tensor
+
+
+@dataclass
+class Inp3:
+    f: torch.Tensor
+    p: torch.Tensor
```

Comment:
what's the difference between Inp2 and Inp3?

Reply:
Nothing other than field names, they just happened to be defined locally which defeats serde, so pulled them up.